Source/WebKit/WebProcess/GPU/graphics/RemoteGraphicsContextGLProxyFunctionsGenerated.cpp

```diff
@@ -3047,17 +3048,25 @@ void RemoteGraphicsContextGLProxy::setDrawingBufferColorSpace(const WebCore::Des
     }
 }
 
+void RemoteGraphicsContextGLProxy::drawingBufferToPixelBuffer(WebCore::GraphicsContextGLFlipY, RefPtr<WebCore::PixelBuffer>&&)
+{
+    RELEASE_ASSERT_NOT_REACHED();
+}
+
 RefPtr<WebCore::PixelBuffer> RemoteGraphicsContextGLProxy::drawingBufferToPixelBuffer(WebCore::GraphicsContextGLFlipY arg0)
 {
     if (isContextLost())
         return { };
-    auto sendResult = sendSync(Messages::RemoteGraphicsContextGL::DrawingBufferToPixelBuffer(arg0));
+    PixelBufferFormat format { AlphaPremultiplication::Unpremultiplied, PixelFormat::RGBA8, DestinationColorSpace::SRGB() };
+    auto pixelBuffer = ShareablePixelBuffer::tryCreate(format, WebCore::IntSize(m_currentWidth, m_currentHeight));
```

Comment:
The caller actually needs a ByteArrayPixelBuffer. This is due to the way WebCore::ImageData works.

I think this patch produces just black, at least in most cases..